flink-table/flink-sql-client/src/test/resources/sql/set.q

```diff
@@ -44,6 +44,14 @@ CREATE TABLE hive_table (
 
 # list the configured configuration
 set;
+execution.attached=true
+execution.savepoint.ignore-unclaimed-state=false
+execution.shutdown-on-attached-exit=false
+execution.target=remote
+jobmanager.rpc.address=localhost
```

Comment:
This may also be different on differet machines.

Reply:
After digging into the code, the 

```
execution.savepoint.ignore-unclaimed-state=false
execution.shutdown-on-attached-exit=false
execution.target=remote
``` 

is set by the `DefaultCli`. 

In `DefaultContext`, it collects avaliable CLI and determine which CLI should be used. In the test, it just uses `DefaultCli`. Then it will convert the CLI to the configuration with specified options in deployment entry. In this way, we put the options list above into the configuration. 

For more details, you can refer to the `DefaultContext#createExecutionConfig`.

The `jobmanager.rpc.address=localhost` is added by MiniCluster. In the test we create a `MiniClusterResource`. In the `MiniClusterResource#startMiniCluster`, it create a `MiniCluster` with configuration. The used configuration that uses the 'localhost' as `jobmanager.rpc.address`. 

Therefore, I think all the options in the list are generated by default, which is consistent on different machines. For the first part,  it is influenced by YAML Deployment Entry. For the second part, it used the `localhost` as the default value.